Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.yml

```diff
@@ -241,25 +241,109 @@ display: Microsoft Defender for Endpoint
 name: Microsoft Defender Advanced Threat Protection
 script:
   commands:
-  - description: Tests connectivity to Microsoft Defender for Endpoint.
-    name: microsoft-atp-test
+  - description: Find machines seen with the requested internal IP address in the time range of 15 minutes prior and after a given timestamp.
+    name: microsoft-atp-list-machines-by-ip
```

Comment:
Next time lets try to keep the same order of the commands for the sake of easy CR :)